app/locales/sv.json

```diff
@@ -44,16 +51,33 @@
   "file": "Fil",
   "dismiss": "AVFÄRDA",
   "lv": "lv",
+  "aboutPage": {
+    "title": "Välkommen till GLAM Wiki Dashboard — en statistikpanel",
+    "subtitle": "Ett verktyg för användningsanalys för GLAM-organisationer att följa fritt material tillgängliggjort på Wikimedias plattformar",
+    "body": "<p>GLAM Wiki Dashboard är ett projekt från Wikimedia Israel, med stöd och vägledning från Wikimedia Sverige, byggt på en infrastruktur av öppen källkod vilken ursprungligen utvecklats av Wikimedia CH. Den syftar till att hjälpa GLAM-organisationer (porträttgallerier, bibliotek, arkiv och museer) att följa användningen av det fria material de laddat upp till Wikimedias plattformar. Verktyget visualiserar statistisk data som reflekterar nivån av av synlighet och användning av dessa fria filer, utifrån flera aspekter och tidsperioder. Ackumulerad data, som presenteras i diverse diagram och grafer, låter organisationerna skaffa insikter, identifiera trender och preferenser, och få grepp på det generella inflytandet detta fria innehåll har på en publik av användare av Wikimediaprojekten världen över. Klicka på länken för att <a href=\"/join\">gå med i GLAM Wiki Dashboard.</a></p><p>Främja allmänhetens medvetenhet om vikten av fri kunskap och expansionen av den kulturella allmänningen är ett av Wikimediarörelsens större mål och en essentiell komponent av Wikimedia Israels uppdrag. För att nå detta mål söker vi att etablera meningsfulla samarbete med innehålls- och kunskapsorganisationer och uppmuntrar dessa att släppa material till den kulturella allmänningen och att ladda upp det på internet. GLAM Wiki Dashboard visualiserar nyttan av att släppa materialet för de deltagande organisationerna och allmänheten.</p><p><strong>Vi hoppas att detta verktyg kommer att öka medvetenheten av vikten om att tillgängliggöra objekt kulturarv och kultur på det globala nätverket och uppmuntrar fler GLAM-organisationer att öppna portarna till sin kunskap genom internet, gratis, till förmån för allmänheten och för att främja tillgången till kunskap, utbildning och fördomsfrihet.</strong></p>",
```

Comment:
meningsfulla samarbete -> samarbeten

Reply:
medvetenheten av vikten om att tillgängliggöra -> medvetenheten om vikten av att..

Reply:
objekt kulturarv och kultur (?) -> det materiella kulturarvet. Var "items of human heritage and culture" på eng

Reply:
på det globala nätverket -> på nätet

Reply:
och ta bort "genom internet", behöver inte upprepas i samma mening